code/modules/client/preferences/admin.dm

```diff
@@ -12,6 +12,21 @@
 
 	return is_admin(preferences.parent) && CONFIG_GET(flag/allow_admin_asaycolor)
 
+/// Whether you want to allow pings in asay to make a sound
+/datum/preference/toggle/asay_pings
```

Comment:
DO NOT BEND THE KNEE

Reply:
if I hear that another admin has their asay pings disabled I'm going to vv them on then ping them repeatedly

Reply:
This is a dumb pref, tbh. Get rid of it.